sherpa/astro/ui/tests/test_astro_ui_plot.py

```diff
@@ -3579,3 +3579,163 @@ def test_set_plot_opt_explicit_astro():
     assert fig.axes[3].get_yscale() == 'linear'
 
     plt.close(fig)
+
+
+@requires_pylab
+@pytest.mark.parametrize("idval", [None, "bob"])
+def test_plot_fit_resid_handles_data_log(idval, clean_astro_ui):
+    """Check that log handling is correct: data=log
+
+    See also test_plot_fit_resid_handles_resid_log.
+
+    I thought we had tests of this, but apparently not.
```

Comment:
I'm not sure that's a useful comment to have in the code, because - as soon as this is merged - it's not correct any longer: We then will have a test (this one). If you want to leave a comment just for me, it might be better to use GH to leave a comment like I do here, so that it does not end up in the commited code?

Reply:
This has been removed in the refactoring,